### PR TITLE
Deactivate project and update README

### PR DESCRIPTION
**Pull request by justinpotts**

### CODE REVIEW

**Comment by stephendonner** [comment]
r+ with s/MDN/Affiliates in https://github.com/justinpotts/Affiliates-Tests/commit/5a9aae5d7e2e6ef393b12aec8a3a576dc989c7a3#commitcomment-15127722 addressed; thanks!

File: README.md [hunk 1]
```diff
@@ -1,139 +1,5 @@
...
-
-    This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0.
-    If a copy of the MPL was not distributed with this file, You can obtain one at
-    http://mozilla.org/MPL/2.0/.
-
-[MPL]: http://www.mozilla.org/MPL/2.0/
+This repository is no longer being actively maintained, and the content has
+been removed. Please use the repository history to access the previous content.
+We are not currently executing or developing tests for Affiliates in this repository.
+You can read more about MDN development and testing in the
```
Comment (stephendonner) [inline README.md]: @justinpotts I think you can just get rid of the "You can read..." and "[project documentation]..." lines entirely, please!

**Comment by stephendonner** [comment]
Thx @justinpotts ! @davehunt: do you know why even with a re-run of https://travis-ci.org/mozilla/Affiliates-Tests/builds/98396802 (and a PASS) Travis is still reporting that the PASSed status isn't being reported?  I also see the notice that this repo is considered a "Legacy Build" and ran on their new platform, via https://blog.travis-ci.com/2015-11-27-moving-to-a-more-elastic-future

**Comment by bobsilverberg** [comment]
I am guessing it is the removal of the ` .travis.yml` file that is now causing Travis to not run tests, but GitHub thinks it is required. I'll see if I can change that so it is no longer required for this repo.

**Comment by bobsilverberg** [comment]
@stephendonner Okay, this can now be merged without requiring Travis to pass first.

**Comment by stephendonner** [comment]
Thx @bobsilverberg !

**Pull request merged**